inference-engine/src/transformations/include/transformations/common_optimizations/wrap_interpolate_into_transposes.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include <transformations_visibility.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API WrapInterpolateIntoTransposes;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief WrapInterpolateIntoTransposes transformation replaces
+ *      Interpolate
+ *  with
+ *      Transpose -> Interpolate -> Transpose
+ *  when
+ *    1) the source Interpolate has the static input rank;
+ *    2) 'axes' input is a Constant;
+ *    3) number of axes is equal to input rank minus 2;
+ *    4) axes contain 0 or 1.
+ *  The reason of this transformation is that now CPU plugin supports interpolation only
```

Comment:
@vladimir-paramuzov can it be useful for GPU as well? Does GPU support NHWC interpolate?

Reply:
According to `inference-engine/src/cldnn_engine/ops/interpolate.cpp`, GPU doesn't support NHWC, because (in 4D case) GPU supports only axes `[0, 1]` or `[2, 3]`.

Reply:
@ilya-lavrenov,  I discussed whether this transformation can be useful in GPU with @vladimir-paramuzov, and he said 'No".

Reply:
Maybe this transformation should be moved to CPU in the separate PR? At this time, there is no place to move tests for the transformation. The place will appear only after the merge of the PR https://github.com/openvinotoolkit/openvino/pull/7488.

Reply:
Discusses with @GlebKazantaev. The transformation will be moved to CPU plugin in the separate PR.